javascript/javascript1/week2/homework.js

```diff
@@ -1,15 +1,53 @@
 // Flight booking fullname function
-
 const getFullname = (firstname, surname, useFormalName) => {
```

Comment:
I know the homework from them like this but just writing in case but always use camelCase in JS `firstName` & `getFullName`